bugbug/models/duplicate.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from imblearn.over_sampling import BorderlineSMOTE
+from sklearn.calibration import CalibratedClassifierCV
+from sklearn.feature_extraction.text import TfidfVectorizer
+from sklearn.pipeline import Pipeline
+from sklearn.preprocessing import LabelEncoder
+from sklearn.svm import LinearSVC
+
+from bugbug import bugzilla, feature_cleanup
+from bugbug.model import BugCoupleModel
+
+
+class super_classifier(CalibratedClassifierCV):
```

Comment:
Nit: rename `super_classifier` to `LinearSVCWithLabelEncoding`

Reply:
We should likely refactor the label encoding stuff in general, but in the future.